contracts/eden/include/members.hpp

```diff
@@ -37,6 +37,10 @@ namespace eden
       std::optional<eosio::public_key> encryption_key;
 
       uint64_t primary_key() const { return account.value; }
+      uint128_t by_representative() const
```

Comment:
Since the election_rank is built into this index, perhaps we should call this `by_group` or something more accurate. It's not really by_representative, right?

Reply:
I don't see why it isn't by representative.  The election_rank is just there to disambiguate.